src/VisualStudio/IntegrationTest/TestUtilities/ScreenshotService.cs

```diff
@@ -26,7 +36,12 @@ public static void TakeScreenshot(string fullPath)
                 var directory = Path.GetDirectoryName(fullPath);
                 Directory.CreateDirectory(directory);
 
-                bitmap.Save(fullPath, ImageFormat.Png);
+                using (var fileStream = new FileStream(fullPath, FileMode.Create))
```

Comment:
Nit: use `FileAccess.Write` vs the implicit `FileAccess.ReadWrite` that is currently present.

Reply:
➡️ Amended second commit to fix